utils/githooks/README.md

```diff
@@ -1,19 +1,18 @@
 # Git hooks
 
-This directory contains githooks that can be employed locally to
-automate things like copyright updates on modified files
+This directory contains a githooks framework that can be
+employed locally to automate things like copyright updates
+on modified files.
+
+It supports adding local hooks by placing your custom hook in
+utils/githooks/<hook>.d/<num>-<name> files.
 
 To use the pre-commit hook here, do the following locally
 
 ```
-
-cat << EOF 2>&1 > .git/hooks/pre-commit
-#!/bin/sh
-
-# Invoke the githook for copyright validation
-exec utils/githooks/pre-commit 1>&2
-EOF
-
-chmod 700 .git/hooks/pre-commit
-
+for d in utils/githooks/*.d; do
+    h=${d%.d};
+    h=${h##*/}; echo $h; done
+    ln -s ../../utils/githooks/$h .git/hooks/$h
+done
```

Comment:
This is wrong, it should be as below.  Note I added a f to the ln command to overwrite what's there.
```suggestion
for d in utils/githooks/*.d
do
    h=${d%.d}
    h=${h##*/}
    echo $h
    ln -sf ../../utils/githooks/$h .git/hooks/$h
done
```